### ISSUE TITLE
Middleware fails

### ISSUE THREAD

**Issue by brized** [rednaks/django-ip-geolocation#36]
Using the middleware in settings.py throws error:

```
File "C:\Program Files\Python37\lib\site-packages\django_ip_geolocation\middleware.py", line 15, in __init__
  super(IpGeolocationMiddleware).__init__(get_response)
TypeError: super() argument 1 must be type, not function
```

**Comment by rednaks** [comment]
thank you for reporting the issue, I'll fix it asap.

**Comment by rednaks** [comment]
@brized the bug is fixed in  v1.5.1 you can update now.

Thank you for your help and let me know if you still have the issue.
